package.json

```diff
@@ -126,15 +126,15 @@
     "prettier": "2.7.1",
     "prompts": "^2.4.2",
     "replace-in-file": "^6.3.5",
-    "rollup-plugin-visualizer": "^5.8.0",
-    "serve": "^14.0.1",
+    "rollup-plugin-visualizer": "^5.8.3",
+    "serve": "^14.1.2",
     "ts-jest": "28.0.7",
     "typescript": "4.9.3",
-    "vite": "^3.0.0",
-    "vite-plugin-checker": "^0.4.9",
+    "vite": "^4.0.1",
```

Comment:
Perhaps it's time to pin these deps? Especially since `@vitejs/plugin-react` is pinned.

Reply:
Done!